Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -43,6 +43,8 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     var bookmarkFolder: BookmarkFolder?
     var refreshControl: UIRefreshControl?
 
+    fileprivate var displayedActionSheet: PhotonActionSheet?
```

Comment:
I think you can dismiss a modal action sheet via `self.presentedViewController` which is a built in property of the VC.